src/main/java/racingcar/controller/Winner.java

```diff
@@ -1,11 +1,13 @@
 package racingcar.controller;
 
+import java.util.ArrayList;
 import java.util.List;
-import java.util.stream.Collectors;
 import racingcar.model.Car;
 
 public class Winner {
 
+  public List<Car> nameOfWinner = new ArrayList<>();
+
   private int getMaxLocation(List<Car> cars) {
     return cars.stream().mapToInt(Car::getLocation).max().orElse(0);
```

Comment:
작성해주신 코드를 스트림을 사용하지 않고 작성해보면 아래와 같이 표현이 됩니다. 
프로그래밍 요구사항에 보시면 
**indent(인덴트, 들여쓰기) depth를 2를 넘지 않도록 구현한다. 1까지만 허용한다.** 라는 요구사항이 있습니다. 
작성해주신 코드에 for문안에 if문이 들어가면서 2dept가 되었네요 확인 부탁드립니다.
```suggestion
      for (Car car : cars) {
          int location = car.getLocation();
          if (!seen || location > best) {
              seen = true;
              best = location;
          }
      }
```